_libraries/ruby-odata4.md

```diff
@@ -0,0 +1,11 @@
+---
+category: other
+name: OData4
+link: https://rubygems.org/gems/odata4
+version: V4
+object: Client
+downloads:
+	- source: Github
```

Comment:
Github has pointed out the inconsistency with tabs and spaces. Can you please update the tabs to spaces?